snapshot/bundlestore/browse_store.go

```diff
@@ -0,0 +1,76 @@
+package bundlestore
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/scootdev/scoot/os/temp"
+)
+
+// Wraps another store and, when reading and when bundle name contains a slash, returns interpreted data.
```

Comment:
I think this is addressed in the next line. The slash is because we can treat that as a reserved character wrt to bundle names and they're also required to represent a path. So it's bundleName+contentPath, which make sense to me. I'll try to clarify a bit.